libs/feature/browser-extension/src/players/media-element-js.ts

```diff
@@ -68,7 +68,7 @@ export class MediaElementJS extends FrameworkPlayer<MeJS.Player> implements A11y
      */
     private adjustFeature(feature: string, enabled: boolean): void {
         for (let player of this.players) {
-            const position = this.getFeaturePosition(player, feature);
+            const position = MediaElementJS.getFeaturePosition(player, feature);
```

Comment:
Why did this change?

Reply:
Was suggested by my IDE, didn't had a proper look at it tbh